### PR TITLE
[TASK] Revise documentation syntax

### PR DESCRIPTION
**Pull request by brotkrueml**
- Remove "coding: utf-8 -*- with BOM" -> in practise it is without BOM
- Use Includes.txt on every page to avoid doubling roles
- Structure headers correctly
- Use "figure" with caption where appropriate
- Use "vendor/bin/typo3" throughout the document for unity
- Adjust .editoconfig: Show line length at 80 chars (e.g. in PhpStorm) and define indentation at 3 chars (like in core documentation)
- git-ignore generated documentation

### CODE REVIEW

**Comment by tomasnorre** [comment]
@brotkrueml Thanks a lot for you PRs, could you please add the Changelog here to or don't you see it needed as isn't not a direct change in code/functionality? but more formatting cleanup?

**Comment by brotkrueml** [comment]
Yup, that was the intention.

> @brotkrueml Thanks a lot for you PRs, could you please add the Changelog here to or don't you see it needed as isn't a direct change in code/functionality? but more formatting cleanup?

**Comment by tomasnorre** [comment]
> Yup, that was the intention.
> 
> > @brotkrueml Thanks a lot for you PRs, could you please add the Changelog here to or don't you see it needed as isn't a direct change in code/functionality? but more formatting cleanup?

Sorry for asking stupidly: 

1) Intention of updating the changelog or 
2) doing formatting cleanup 

:)

**Comment by brotkrueml** [comment]
The user has no benefit from an entry in the changelog, imho. But I can create one, if you like.

**Comment by tomasnorre** [comment]
No that's fine, just wanted to be sure what you meant :) Will merge.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst [hunk 1]
```diff
@@ -1,32 +1,20 @@
...
 A "cron-job" refers to a script that runs on the server with time
 intervals.
 
 For this to become reality you must ideally have a cron-job set up.
 This assumes you are running on Unix architecture of some sort. The
-crontab is often edited by "crontab -e" and you should insert a line
+crontab is often edited by :shell:`crontab -e` and you should insert a line
 like this:
 
 ::
```
Comment (tomasnorre) [inline Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst:16]: I wonder why this changed have it looks.
https://docs.typo3.org/p/aoepeople/crawler/master/en-us/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.html

It like the coloring in the current docs.

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst [hunk 2]
```diff
@@ -64,7 +54,7 @@ If you run it you will see a list of options which explains usage.
 
          **Examples:**
 
-         ::
+         .. code-block:: text
```
Comment (tomasnorre) [inline Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst:57]: also like the old formatting better.
https://docs.typo3.org/p/aoepeople/crawler/master/en-us/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.html

File: Documentation/UseCases/CacheWarmup/Index.rst [hunk 3]
```diff
@@ -1,75 +1,87 @@
...
+   how will we execute this from CLI during deployment? I don't know which
+   deployment tool you use, but it's not important as long as you can execute
+   shell commands. What would you need to execute?
 
+   ::
 
-The last step will add the pages to the queue, and you would need a scheduler task setup to have them executed. Go to
-the scheduler module and do following steps:
+       # Done to make sure the crawler queue is empty, so that we will only crawl important pages.
+      $ vendor/bin/typo3 crawler:flushQueue all
```
Comment (tomasnorre) [inline Documentation/UseCases/CacheWarmup/Index.rst:50]: The identation is one off here.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst:16]: It looks now like this:
image
But I can revert that change.

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst:57]: Before it was rendered with the default highlighting (which was PHP). Now it is rendered like a - well - text.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst (hunk 1 above)
Comment (tomasnorre) [reply Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst:16]: Sorry meant the link below, this view is a little hmm for review.

Comment (tomasnorre) [reply Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst:16]: It's the :: block

File: Documentation/UseCases/CacheWarmup/Index.rst (hunk 3 above)
Comment (brotkrueml) [reply Documentation/UseCases/CacheWarmup/Index.rst:50]: Yes, changed it.

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst (hunk 2 above)
Comment (tomasnorre) [reply Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst:57]: I know but I sort of like the highlighting, that the reason I chose the default.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst [hunk 4]
```diff
@@ -1,32 +1,20 @@
...
 
-   * * * * * [pathToYourTYPO3Installation-composer-bin-dir]/typo3cms crawler:buildQueue <startpage> <configurationKeys> > /dev/null
+   * * * * * vendor/bin/typo3 crawler:buildQueue <startpage> <configurationKeys> > /dev/null
```
Comment (tomasnorre) [inline Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst:17]: This was the line i meant.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst:16]: Hmm, don't understand. The "::" is untouched to the previous version.

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst]: Changed it, also the default highlighting back to PHP.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst (hunk 1 above)
Comment (tomasnorre) [reply Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst:16]: That's also what I don't get. Ignore it for now. It's not that important.

**Comment by tomasnorre** [comment]
Will merge when CI is done.

**Pull request merged**